chat-plugins/github.js

```diff
@@ -0,0 +1,94 @@
+/*
+	Github plugin by Spandamn
+	This is based on https://github.com/Ecuacion/Pokemon-Showdown-Node-Bot/blob/master/features/github/index.js
+	Yes, some of the code is copied so credits to xfix, Ecuacion, jd, and anyone who's name I forgot to put here
+*/
+'use strict';
+
+let config = {};
+
+if (!Config.github) {
+	return;
+} else if (Config.github === {}) {
+	config.port = 3420;
```

Comment:
I'm not a lawyer, but I think by "hosting" it means production-level hosting.  But I don't understand why this is a feature that someone would want to use on a "testing server" where we should make a comment specifically in that case for if their using C9.  It that is the case, then they should already know what ports they have to use where we shouldn't have to document it, just as we don't document it for starting the server.